diff.py

```diff
@@ -257,33 +257,35 @@ def complete_symbol(
     )
     parser.add_argument(
         "-0",
-        "--zeroway",
-        dest="zeroway",
-        action="store_true",
+        "--diff_type=single_base",
+        dest="diff_type",
+        action="store_const",
+        const="single_base",
         help="""View the base asm only (not a diff).""",
     )
     parser.add_argument(
         "-1",
-        "--oneway",
-        dest="oneway",
-        action="store_true",
+        "--diff_type=single",
+        dest="diff_type",
+        action="store_const",
+        const="single",
         help="""View the current asm only (not a diff).""",
     )
     parser.add_argument(
         "-3",
-        "--threeway=prev",
-        dest="threeway",
+        "--diff_type=3prev",
```

Comment:
not a fan of the snake casing, internal-looking "3prev" token, and breaking backward compat by changing this